src/com/reco1l/andengine/Entities.kt

```diff
@@ -40,6 +40,34 @@ var ExtendedEntity.size
         height = value.y
     }
 
+/**
+ * The draw size of the entity.
+ */
+val ExtendedEntity.drawSize
+    get() = Vec2(drawWidth, drawHeight)
+
+/**
+ * The position of the entity.
+ */
+var ExtendedEntity.position
+    get() = Vec2(x, y)
+    set(value) {
+        x = value.x
+        y = value.y
+    }
```

Comment:
Can't this call `setPosition` to avoid an extra container-child propagation?

Reply:
Yes, sure.